src/script/processTarget/setWebpack/setEnv.js

```diff
@@ -0,0 +1,4 @@
+module.exports = (path) => {
+  process.env.NODE_ENV = 'development'
+  process.env.NODE_PATH = path
```

Comment:
I agree with the first point.
I deasgree with the second: maybe some (targeted) projects will use this NODE_PATH environment variable.

So I'll let it that way